src/System.Console/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,2 +1,28 @@
-<!-- Not a bug, this API was recently added to the allowed list -->
-user32.dll!GetKeyState
+kernel32.dll!Beep
+kernel32.dll!FillConsoleOutputAttribute
+kernel32.dll!FillConsoleOutputCharacterW
```

Comment:
You're right, fixed and removed the baseline. (They may need to bring it back when they implement it..)